infrastructure/server-setup/playbook-3.yml

```diff
@@ -197,6 +220,32 @@
         group: 1000
         owner: 1000
         mode: ugo+rwx
+      when: not elasticsearch_data_backup.stat.exists
+
+    - name: Check influxdb data directory
+      stat:
+        path: /data/influxdb
+      register: influxdb_data
+
+    - name: 'Create influxdb data directory'
+      file:
+        path: /data/influxdb
+        state: directory
+      when: not influxdb_data.stat.exists
+
+    - name: Check influxdb data backup directory
+      stat:
+        path: /data/backups/influxdb
+      register: influxdb_data_backup
+
+    - name: 'Create influxdb backup directory'
+      file:
+        path: /data/backups/influxdb
+        state: directory
+        group: 1000
+        owner: 1000
+        mode: g+rwx
+      when: not influxdb_data_backup.stat.exists
```

Comment:
Did we not create influxdb data directory previously? If we did then this might become duplicate because I don't see anything in the left side of the changes unlike the `playbook-1.yml`

Reply:
@Zangetsu101 for some reason it is done in the docker-manager-first only.  Maybe there is no need to create these directories in each node, but I am scared to change it

Reply:
I removed the block in docker-manager-first as the rest of these changes are already in develop with the exception of the version number bump